src/rust/ensogl/lib/core/src/display/scene.rs

```diff
@@ -458,9 +458,12 @@ impl Dom {
 #[derive(Clone,CloneRef,Debug)]
 pub struct DomLayers {
     /// Back DOM scene layer.
-    pub back: DomScene,
+    pub back : DomScene,
+    /// Back DOM with fullscreen visualization. Kept separately from `back`, because use different
+    /// Camera
```

Comment:
this has several problems:
1. It's not gramatically correct ('use')
2. does not have dot at the end.
3. States that this is a "back DOM", which is not specific enough, as this is a back DOM **scene**.
4. Does not explain what a "different camera" is and why do we use a different camera here. It should be explained here soduring future refactoring it would be clear what it is about.